src/r5py/r5/regional_task.py

```diff
@@ -227,6 +227,14 @@ def destinations(self):
     def destinations(self, destinations):
         self._destinations = destinations
 
+        # R5 has a maximum number of destinations for which it returns detailed
+        # information, and it’s set to 5000 by default.
+        # The value is a static property of com.conveyal.r5.analyst.cluster.PathResult;
+        # static properites of Java classes can be modified in a singleton kind of way
+        com.conveyal.r5.analyst.cluster.PathResult.maxDestinations = (
+            len(destinations) + 1
```

Comment:
This could lead to problems down the road, when running/preparing multiple analyses in parallel. Maybe a `max(old value, new value)` would be smart